### DISCUSSION TITLE
Network name missing

### DISCUSSION THREAD

**Discussion by ceejayemm** [rern/rAudio#1657] [Fixed Issues]
I think this minor problem occurred in one of the most recent releases (I am currently running 20240406).  On the Networks page when using a wired connection the Wired LAN network name is shown as 'undefined'.  Clicking on the Info button shows:

# ifconfig enu1u1
enu1u1: flags=4163  mtu 1500
        inet 192.168.86.31  netmask 255.255.255.0  broadcast 192.168.86.255
        ether b8:27:eb:88:f7:25  txqueuelen 1000  (Ethernet)

![image](https://github.com/rern/rAudio/assets/25418025/f9c1e354-0b90-45c9-94e3-74775c2e1202)

It does not seem to affect wifi networks as these are still shown correctly.

As I say only a minor issue.

Chris

**Comment by R101G** [comment]
Same here.

**Comment by ceejayemm** [comment]
@rern 
Thanks, update 20240408 goes some way to fixing this issue. The word 'undefined' has been replaced by the network IP Number but not the Network Name.  It used to be, for example, 

CMHomeNET : 192.168.86.31 : 192.268.86.1 (as per the current WiFi section) 

but is currently: 

192.168.86.31 : 192.168.86.1

Regards.

Chris
